### DISCUSSION TITLE
ラベルとマイルストーンで進捗を管理できるように試行錯誤開始

### DISCUSSION THREAD

**Discussion by pss-aileen** [first-contributions-ja/first-contributions-ja.github.io#111] [【運営】相談・報告]
### Label
- 親タスク
- 小タスク
- 単独タスク
を作成

### Milestones
- 親タスク、小タスクがあるものはマイルストーンでグルーピング

**Comment by pss-aileen** [comment]
![スクリーンショット 2024-03-25 1 10 41](https://github.com/first-contributions-ja/first-contributions-ja.github.io/assets/140990100/c8b82453-a608-4423-9272-3e339ae8d097)

![スクリーンショット 2024-03-25 1 10 50](https://github.com/first-contributions-ja/first-contributions-ja.github.io/assets/140990100/87ad30e4-b877-4976-b538-33f44f46c005)

**Comment by kazzyfrog** [comment]
第４回のMTGより

issueは、以前のように「優先度高いタスクのまとめ」を、
１つピン留めする方式の方がいいかも？と感じたので、戻してみようかと思ってます！

ピン留めするまとめ以外のissueは、親子関係なく全部並列な感じで！

**Reply by kazzyfrog** [reply]
なので、問題なければ、
一旦「親タスク・小タスク・単独タスク」のラベルを剥がしてしまっても良いですかね。。！？

**Reply by kazzyfrog** [reply]
現在最優先のタスク達と、その他！、という感じでシンプルにするイメージです！

**Reply by pss-aileen** [reply]
全く問題ありません！
ありがとうございます！！

**Reply by kazzyfrog** [reply]
了解です！！
ありがとうございます！🙌
